scipy/special/__init__.py

```diff
@@ -852,3 +852,12 @@ def btdtri(*args, **kwargs):  # type: ignore [no-redef]
 
 
 btdtri.__doc__ = _ufuncs.btdtri.__doc__  # type: ignore [misc]
+
+
+def _get_include():
+    """This function is for development purposes only.
+
+    This function could disappear or its behavior could change at any time.
+    """
+    import os
```

Comment:
Just curious, why `import os` in the method itself and not the file? Is this best practice for SciPy?

Reply:
It’s not good practice. I just wanted to minimize the footprint of the function; although here it’s mostly for show I guess.

Reply:
In `__init__.py` files, we need to minimize accidental exposure of names. We don't want `scipy.special.os` to be a thing.

Reply:
Yes. That’s right. Thanks @rkern. I knew I’d done it for a reason.